code/__DEFINES/sound.dm

```diff
@@ -1,3 +1,8 @@
 //max channel is 1024. Only go lower from here, because byond tends to pick the first availiable channel to play sounds on
 #define CHANNEL_LOBBYMUSIC 1024
 #define CHANNEL_ADMIN 1023
+#define CHANNEL_VOX 1022
+
+//THIS SHOULD ALWAYS BE THE LOWEST ONE!
+//KEEP IT UPDATED
+#define CHANNEL_HIGHEST_AVAILABLE 1021
```

Comment:
this is gross but as long as these channels are defines there's not much in the way of a better way to do it